services/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "buddy-app-services",
+  "version": "1.0.0",
+  "description": "Buddy app graphql serverless service",
+  "scripts": {
+    "start": "serverless offline",
+    "prisma-generate": "prisma generate",
+    "deploy": "serverless deploy",
+    "remove": "serverless remove",
+    "build": "serverless package",
+    "create_domain": "serverless create_domain",
```

Comment:
Maybe you could follow the convention using dash (`-`) for scripts

```
    "create-domain": "serverless create_domain",
    "delete-domain": "serverless delete_domain"
```
Just like it's defined here:
https://github.com/BuildIt-Poland/buddy-app/blob/ffe929d31dfa9a822c729e322ea0d8dea5660bbe/services/package.json#L7

Reply:
fixed 2ed233f